appinventor/appengine/src/com/google/appinventor/client/editor/simple/ComponentDatabase.java

```diff
@@ -26,67 +25,10 @@
  */
 class ComponentDatabase implements ComponentDatabaseInterface {
 
-  /*
```

Comment:
#Explaining the Major changes, I feared doing but eventually had to do.

I moved the Component from this ComponentDatabase class to the ComponentDatabaseInterface.
I also renamed it to the class name to ComponentDefintion.